src/central_control/scripts/aabb.py

```diff
@@ -0,0 +1,58 @@
+from math import sqrt
+from point import Point
+from range import Range
+
+class AABB:
```

Comment:
Can you put a short comment on what AABB stands for

Reply:
Added comment